applications/nrf5340_audio/src/drivers/CMakeLists.txt

```
@@ -4,6 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-target_sources(app PRIVATE
-	       ${CMAKE_CURRENT_SOURCE_DIR}/cs47l63_comm.c
+target_sources_ifdef(CONFIG_HW_CODEC_CIRRUS_LOGIC
```

Suggest we also rename:
```suggestion
target_sources_ifdef(CONFIG_HW_CODEC_CS47L63
```

---

I concur

---

You mean to rename `NRF5340_AUDIO_CS47L63_DRIVER` to `CONFIG_HW_CODEC_CS47L63`?

Anyway this line should definitely be checking `NRF5340_AUDIO_CS47L63_DRIVER` and not `HW_CODEC_CIRRUS_LOGIC`, will update